src/linear_solver_logs/solve_log_ma.jl

```diff
@@ -75,14 +100,17 @@ A mutable structure that stores information about a randomized linear solver's b
 - `iterations::Int64`, the number of iterations of the solver.
```

Comment:
The collection rate part needs to be made explicit.

Reply:
I made this say the current iteration of the solver because it was unclear. I also clarified the collection rate description